specifyweb/frontend/js_src/lib/components/SchemaConfig/UniquenessRules.tsx

```diff
@@ -128,22 +132,16 @@ export function TableUniquenessRules({
   );
 
   return (
-    <Dialog
-      buttons={
-        <>
-          <Button.DialogClose>{commonText.close()}</Button.DialogClose>
-          {saveBlocked ? (
-            <Button.Danger
-              className="cursor-not-allowed"
-              onClick={(): void => console.error(tableRules)}
-            >
-              {icons.exclamation}
-              {commonText.save()}
-            </Button.Danger>
-          ) : (
+    <Form id={formId}>
```

Comment:
form looks good, but it's in a wrong place - can you please put it inside the `<Dialog>`?

explanation:
while, looking at JSX, it might look as if Form contains all of the content
BUT in reality, since Dialog is rendering using react portal (meaning, the actual dom nodes are attached in a different place), the `<form>` element is actually empty if you look at the DevTools - this is not right as the form should contain all the elements (or the elements must use the `form` attribute, like the submit button does)